tox.ini

```diff
@@ -37,7 +41,7 @@ commands =
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage
 passenv =
-    *
```

Comment:
What does this change do?

Reply:
That was a mistake. I kinda want coveralls to only run on one environment